docs/source/release/index.rst

```diff
@@ -12,6 +12,7 @@ Release Notes
    These release notes are for versions of ibis **1.0 and later**. Release
    notes for pre-1.0 versions of ibis can be found at :doc:`release-pre-1.0`
 
+* :feature:`2938` Add `__getstate__` and `__setstate__` methods to Node
```

Comment:
can you just say, that serialization-deserialization via pickle is now byte compatible between different processes.